deploy/cro-aws-config.yml

```diff
@@ -4,6 +4,6 @@ metadata:
   name: cloud-resources-aws-strategies
 data:
   _network: >-
-    { "production": { "createStrategy": { "CidrBlock": "10.1.0.0/23" } } }
+    { "production": { "createStrategy": {} } }
```

Comment:
i'm removing this because cro should now default. this was included previously to allow for development in a non add-on scenario. For dev purposes, we should allow cro to use it's default logic. A dev can specify an entry here if they wish.